frontend/packages/dev-console/integration-tests/features/virtulization/virtualMachines-page.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Virtual Machines page
+    As a user, I should be able visit Vitual Machines page
+
+
+    Background:
+        Given user has installed OpenShift Virtualization Operator
+        And user has created the OpenShift Virtualization Deployment
+        And user has created the HostPathProvisioner Deployment
+        And user is at developer perspective
+        And user has selected namespace "aut-virtualization"
+        
+
+    @regression, @smoke, @cnv
+    Scenario: Virtual Machines Card on +Add page: VM-01-TC03
+        Given user is at Add page
+        Then user will see Virtual Machines Card on Add page
+
+    
+    @smoke, @cnv
+    Scenario: Virtual Machines on Developer Catalog page
+        Given user is at Developer Catalog page
+        Then user will see Virtual Machines type
+    
+
+    @smoke, @cnv
+    Scenario: Virtual Machines page - Empty View
+        Given user is at Developer Catalog page
+        When user clicks on Virtual Machines type
+        Then user will see "No Catalog items found"
```

Comment:
If the user has created OpenShift Virtualization Deployment and HostPathProvisioner Deployment how can it be empty

Reply:
Creating these two deployments won't create any templates